Basic-Audio-Transformer/README.md

```diff
@@ -67,6 +67,38 @@ const transformStream = async (publisher) => {
 };
 ```
 
+### Publisher.setAudioMediaProcessorConnector() method
+The OpenTok.js `Publisher.setAudioMediaProcessorConnector()` method applies an audio transformer to a published stream:
+
+```javascript
+publisher
+  .setAudioMediaProcessorConnector(mediaProcessorConnector)
+```
+
+### Audio Transformer
+A Transformer is an object or class instance representing the transformer. For a definition, see the `transformer` parameter of the [`TransformStream()`](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream/TransformStream#parameters) constructor. Also see the [TypeScript Transformer interface](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L1768).
```

Comment:
The line number anchor in that TypeScript definition page could change. Maybe it would be better to write:

```
Also see the definition for `interface Transformer` in the [TypeScript Transformer interface](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts).
```

Reply:
Ahhh, true. Thanks!